### DISCUSSION TITLE
Converting adi files to CSV - easier to work with

### DISCUSSION THREAD

**Discussion by ZB2ham** [N0BOY/FT8CN#56] [Show and tell]
ADI files are all well and good, used by many electronic logging programs - but if you prefer to see a list of contacts as per an old paper logbook, you might like to work with a "friendlier" file format. I wanted to be able to make a logbook in Excel, which of course by the very nature of the program would remain searchable etc. 

After much looking around, the good folk at LOTW put me in touch with a chap who pointed me to a website, which provided a very simple but very effective ADIF-to-CSV converter. You can find this at;
https://ad1c.us/software/ADIF_to_CSV/index.html
and you can see other tools available at that website.

Once converted to a CSV file, you can use macros within Excel to sort your columns into your preferred order, apply formatting etc.

**Comment by Wirezfree** [comment]
Another very useful application to manage and export adif files is "ADIF Master"
It also can "Check" an ADIF File for any errors, it is linked to the official ADIF Specification.
[https://www.dxshell.com/adif-master.html](url)
